pkg/cmd/roachtest/tests/restore.go

```diff
@@ -633,6 +646,11 @@ func (hw hardwareSpecs) String(verbose bool) string {
 	var builder strings.Builder
 	builder.WriteString(fmt.Sprintf("/nodes=%d", hw.nodes))
 	builder.WriteString(fmt.Sprintf("/cpus=%d", hw.cpus))
+	if hw.mem == spec.High {
```

Comment:
nit: could we instead add a `.String()` method to the `nodeHighMemOption` type?

Reply:
mmm, interesting.. but that would print something like high/low/standard/auto and for the restore tests I only want high and low mem. unless you had something else in mind?

Reply:
I was imagining something like:

```
func (mem nodeHigheMemOption) String() {
 switch mem {
   case auto
      return "auto"
   case low
       return "low"
       ...
 }
}
```

Then in the restore roachtest code:
```
if hw.mem != spec.Auto {
 builder.WriteString(fmt.Sptrintf("/%smem",fw.mem.String()))
}
```

I think this pushes more logic away from the restore code and also ensures that if someone picks "standard", which is not default, it's included in the roachtest name.